--- boot.cmd    2020-03-13 18:46:16.053633706 -0700
+++ boot.new    2020-04-06 15:37:20.426513093 -0700
@@ -3,6 +3,13 @@
 # Please edit /boot/armbianEnv.txt to set supported parameters
 #

+# Save the current devnum (bootcmd_usb0 changes it)
+setenv preusb_devnum "${devnum}"
+# Attempt USB boot if present (adds 7 seconds to boot)
+run bootcmd_usb0
+# Change devum back
+setenv devnum "${preusb_devnum}"
+
 setenv load_addr "0x39000000"
 setenv overlay_error "false"
 # default values
